src/types/config.js

```diff
@@ -19,6 +15,9 @@
  * @prop {boolean} [allowFlagging]
  * @prop {boolean} [allowLeavingGroups]
  * @prop {boolean} [enableShareLinks]
+ * The `onXXX` functions may be set by the embedder of the client. The
```

Comment:
I think this might have the unintended consequence of causing TS to attach these comments all to the `onLoginRequest` prop, when eg. VS Code is presenting documentation for a symbol.